TestRx/Model /Mappable/Breed.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Breed.swift
+//  TestRx
+//
+//  Created by Esteban Arrua on 4/10/18.
+//  Copyright © 2018 Rocket League. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class Breed: Mappable {
+    
+    var name: String
+    
+    required init?(json: JSON?) {
+        name = json?.string ?? ""
```

Comment:
here I think there is an error because we're just accessing json?.string which would answer the entire string, so we need to access the 'json?['name'].string' probably, and also I'd add something like this:
`guard let name = json?['name'] else {
       return nil
}`